storage/client/queries/queries.go

```diff
@@ -257,7 +257,8 @@ const (
 		OFFSET $2::bigint`
 
 	Epoch = `
-		SELECT id, start_height, end_height
+		SELECT id, start_height,
+			(CASE id WHEN (SELECT max(id) FROM chain.epochs) THEN NULL ELSE end_height END) AS end_height
```

Comment:
Optional:
To simplify the code and bring it more in line with other similar queries, we could abandon the `Epoch` SQL query and instead expand `Epochs` with `WHERE ($1::bigint IS NULL OR $1::bigint = id)`.

Reply:
Updated 👍